docs/docs/quickstart.mdx

```diff
@@ -4,61 +4,42 @@ hide_title: false
+it assumes that you have [Docker](https://www.docker.com),
+[docker-compose](https://docs.docker.com/compose/), and
+[git](https://git-scm.com/) installed.
+
+:::caution
+While we recommend using `docker-compose` for a quick start in a sandbox-type
+environment and for other development-type use cases, **we
+do not recommend this setup for production**. For this purpose please
+refer to our
+[Installing on Kubernetes](/docs/installation/running-on-kubernetes/)
+page.
+:::
 
 ### 1. Get Superset
-To get started, set the `SUPERSET_VERSION` environment variable with the latest Superset version.
-[Click here](https://github.com/apache/superset/releases) to check the latest version.
 
-```
-$ export SUPERSET_VERSION=<latest_version>
+```bash
+$ git clone https://github.com/apache/superset
 ```
 
-Pull the Superset image from Docker Hub:
+### 2. Start the latest official release of Superset
 
 ```
-$ docker pull apache/superset:$SUPERSET_VERSION
-```
+# Enter the repository you just cloned
+$ cd superset
 
-### 2. Start Superset
-:::tip
-Note that some configuration is mandatory for Superset in order to start. In particular, Superset will not start without
-a user-specified value of `SECRET_KEY` in a Superset configuration file or `SUPERSET_SECRET_KEY` as an environment variable.
-Please see [Configuring Superset](/docs/installation/configuring-superset/) for more details.
-:::
+# Fire up Superset using docker-compose
+$ docker-compose -f docker-compose-image-tag.yml up
 ```
-$ docker run -d -p 8080:8088 \
-             -e "SUPERSET_SECRET_KEY=$(openssl rand -base64 42)" \
-             -e "TALISMAN_ENABLED=False" \
-             --name superset apache/superset:$SUPERSET_VERSION
-```
-
-### 3. Create an account
-```
-$ docker exec -it superset superset fab create-admin \
-              --username admin \
-              --firstname Admin \
-              --lastname Admin \
-              --email admin@localhost \
-              --password admin
-```
-
-### 4. Configure Superset
-```
-$ docker exec -it superset superset db upgrade &&
-         docker exec -it superset superset load_examples &&
-         docker exec -it superset superset init
-```
-:::tip
-This step can take some time. While you wait, feel free to join the official Slack channel to check for new releases,
-ask questions, and engage with the community.
-[Click here to join.](http://bit.ly/join-superset-slack)
-:::
+This may take a moment as docker-compose will fetch the underlying
+docker images and will load up some examples. Once all containers
+are downloaded and the output settles, you're ready to log in.
 
-### 5. Start using Superset
-After configuring your fresh instance, head over to [http://localhost:8080](http://localhost:8080) and
-log in with the default created account:
+### 3. Log into Superset
+Now head over to [http://localhost:8080](http://localhost:8080) and log in with the default created account:
```

Comment:
Superset doesn't start at port 8080, instead it starts at 8088 while using the docker-compose-image-tag.yml file. https://github.com/apache/superset/blob/349e496da519f3462aa5cff2d1bd76490b04fe5d/docker-compose-image-tag.yml#L52

Reply:
good catch!